README.md

```diff
@@ -1,2 +1,30 @@
-# studying-ts-automation-process
-My own repo, my own tests
+**Project**: studying-ts-automation-process
+
+**Project setup**
+- Visual Studio code
+- Node.js
+- TypeScript
+- Cypress
+//will add here some new points later
+
+**How to run the project**
+- Create a new folder for your project on your local PC
+- Cloned the repository from Github https://github.com/AlexandraPopovaz/studying-ts-automation-process.git
+- Run **git init** (create your local repo)
+- Run **npm init** (create a Package.json file)
```

Comment:
Same here. Your project has been initialized already, you need to install dependencies using `npm install` command. That's the purpose of `package.json`: to save/share dependencies.